addons/vehicles/script_component.hpp

```diff
@@ -35,4 +35,9 @@
 
 #define GREEN_NET_TEXTURE "A3\Armor_F\Data\camonet_nato_green_co.paa"
 #define CSAT_NET_TEXTURE "A3\Armor_F\Data\camonet_csat_hex_green_co.paa"
-#define AAF_CAGE_TEXTURE "A3\Armor_F\Data\cage_aaf_co.paa"
\ No newline at end of file
+#define AAF_CAGE_TEXTURE "A3\Armor_F\Data\cage_aaf_co.paa"
+
+/// Transport Items macros ///
+#define MAG_XX(var0,var1) class _xx_##var0 { magazine = QUOTE(var0); count = var1; }
```

Comment:
These already exist in addons\main\script_macros but you're welcome to rename those and refactor them elsewhere if you want to use this syntax instead

Reply:
Oh, I'll just use the base ones then. I didn't think to check that they were already defined, I should look at that file more often.